modules/blocks/process.js

```diff
@@ -139,18 +139,17 @@ Process.prototype.getCommonBlock = function (peer, height, cb) {
  * @method loadBlocksOffset
  * @param  {number}   limit Limit amount of blocks
  * @param  {number}   offset Offset to start at
- * @param  {boolean}  verify Indicator that block needs to be verified
  * @param  {Function} cb Callback function
  * @return {Function} cb Callback function from params (through setImmediate)
  * @return {Object}   cb.err Error if occurred
  * @return {Object}   cb.lastBlock Current last block
  */
-Process.prototype.loadBlocksOffset = function (limit, offset, verify, cb) {
```

Comment:
We removed `verify` parameter, is it related with https://github.com/LiskHQ/lisk/issues/1868? If yes - please remove unused logic.

Reply:
We removed `verify` parameter because now its not optional. We will verify blocks during snapshotting in any case.